### PR TITLE
✨ Add audio save and playback function

### PR DESCRIPTION
**Pull request by Emeraldb01**
## ✅ Checklist

- [x] I have confirmed this pull request does not contain any sensitive information (e.g. passwords, access tokens, etc.)

- [x] The title of this pull request is a short description of the requested changes and conforms to the [Gitmoji](https://gitmoji.carloscuesta.me/) style guide
- [x] I have added a description of the changes I have made below

## 📝 Description

1. Save mp3 streaming audio bytes. At the end of recording, convert bytes to mp3.
2. Playback mp3.

## ✏️ Additional Information

- During save: mp3 file name is specified at the opening of WebSocket (_audio_file_name_)
- During playback: mp3 file name is specified GET parameter
- _AUDIO_DIRECTORY_: Path for all audio file
- _TEMP_MIME_AUDIO_PATH_: Path for temporary audio bytes

### CODE REVIEW

File: src/notice_api/playback/routes.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    try:
+        with open(audio_file_path, "rb") as audio_file:
+            audio_data = audio_file.read()
+    except FileNotFoundError:
+        logger.error(f"Audio file '{filename}' not found")
+        error_message = {"detail": f"Audio file '{filename}' not found"}
+        return JSONResponse(status_code=404, content=error_message)
+
+    logger.info(f"Retrieved audio file: {filename}")
+    return Response(content=audio_data, media_type="audio/mpeg")
```
Comment (alan910127) [inline src/notice_api/playback/routes.py]: 好像也可以用 `FileResponse` 處理，可能就不用自己讀檔了~ 不過可以之後再改

File: src/notice_api/transcript/routes.py [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 from fastapi.responses import HTMLResponse
+from pydub import AudioSegment
 
 from notice_api.core.config import settings
 
 router = APIRouter()
 
+AUDIO_DIRECTORY = "../audio_temp"  # Path for all audio file
+TEMP_MIME_AUDIO_PATH = "../audio_temp/test_audio.bin"  # Path for temporary mime file
+audio_file_name = "note_name"  # Name for saving current audio
```
Comment (alan910127) [inline src/notice_api/transcript/routes.py:16]: 這裡之後應該都會需要改成動態生成的對嗎?

Comment (Emeraldb01) [reply src/notice_api/transcript/routes.py:16]: 對～～🎃🎃

File: src/notice_api/playback/routes.py (hunk 1 above)
Comment (Emeraldb01) [reply src/notice_api/playback/routes.py]: updated!! 🌟🙆

**Review by alan910127 — APPROVED** [review]

**Pull request merged**